src/cca.jl

```diff
@@ -336,3 +340,113 @@ function fit(::Type{CCA}, X::AbstractMatrix{T}, Y::AbstractMatrix{T};
 
     return M::CCA
 end
+
+abstract type MultivariateTest <: HypothesisTest end
+
+struct WilksLambda <: MultivariateTest
+    stat::Float64
+    fstat::Float64
+    df1::Float64
+    df2::Float64
+end
+
+struct LawleyHotelling <: MultivariateTest
+    stat::Float64
+    fstat::Float64
+    df1::Float64
+    df2::Float64
+end
+
+struct PillaiTrace <: MultivariateTest
+    stat::Float64
+    fstat::Float64
+    df1::Float64
+    df2::Float64
+end
+
+function pvalue(ct::MultivariateTest)
+    return 1 - cdf(FDist(ct.df1, ct.df2), ct.fstat)
```

Comment:
It's generally preferable to use `ccdf` rather than `1 - cdf(..)` since the former may sometimes have a more direct or numerically precise implementation.
```suggestion
    return ccdf(FDist(ct.df1, ct.df2), ct.fstat)
```

Reply:
Done